src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass.cs

```diff
@@ -105,6 +119,80 @@ public override BoundNode VisitTryStatement(BoundTryStatement node)
             return null;
         }
 
+        protected Optional<TLocalState> _tryState;
+
+        private void VisitTryBlockWithUnassignments(BoundStatement tryBlock, BoundTryStatement node, ref TLocalState tryState)
+        {
+            if (trackUnassignments)
+            {
+                Optional<TLocalState> oldTryState = _tryState;
+                _tryState = AllBitsSet();
+                VisitTryBlock(tryBlock, node, ref tryState);
+                var tts = _tryState.Value;
```

Comment:
>tts [](start = 20, length = 3)

nit: even though this is moving existing code, consider giving `tts` and `ots` proper names.